custom/icds_reports/models.py

```diff
@@ -1060,3 +1066,51 @@ class Meta:
         app_label = 'icds_model'
         managed = False
         db_table = 'ucr_table_name_mapping'
+
+
+@architect.install(
+    'partition',
+    type='range',
+    subtype='date',
+    constraint='month',
+    column='time_of_use_start',
+    db=get_icds_ucr_db_alias()
+)
+class ICDSAuditEntryRecord(models.Model):
```

Comment:
Can you use a `UUIDField` as the primary key? We've had a table overflow integers and I think that at some point this table would do the same: https://docs.djangoproject.com/en/2.0/ref/models/fields/#uuidfield